images/php/README.md

```diff
@@ -8,33 +8,43 @@
 ---
 <!--monopod:end-->
 
-Minimalist PHP images for building and running PHP applications (CLI).
+Minimalist Wolfi-based PHP images for building and running PHP applications. Includes both `cli` and `fpm` variants.
 
 - [Documentation](https://edu.chainguard.dev/chainguard/chainguard-images/reference/php)
-- [Getting Started Guide](https://edu.chainguard.dev/chainguard/chainguard-images/reference/php/getting-started/)
+- [Getting Started Guide](https://edu.chainguard.dev/chainguard/chainguard-images/reference/php/getting-started-php/)
 - [Provenance Information](https://edu.chainguard.dev/chainguard/chainguard-images/reference/php/provenance_info/)
 
-## Usage 
+## Image Variants 
 
-To try out the image, run:
+Our `latest` tags use the most recent build of the [Wolfi PHP](https://github.com/wolfi-dev/os/blob/main/php.yaml) package. The following tagged variants are available without authentication:
+
+- `latest`: This is a distroless image for running command-line PHP applications. It does not include Composer or busybox, so no shell will be available.
+- `latest-dev`: This is a development / builder image that includes Composer and busybox (not distroless).
```

Comment:
```suggestion
- `latest-dev`: This is a development / builder image that includes Composer and busybox.
```

I don't know if we have a clear enough definition of "distroless", but to me it means "doesn't include a package manager" -- i.e., if this had `apk-tools`, I'd say it was "not distroless", but even then I'm not sure the word is defined enough that people wouldn't disagree.

In any case, let's just drop the mention entirely 😄